src/gui/systray.h

```diff
@@ -21,6 +21,7 @@
 #include "tray/usermodel.h"
 
 #include <QQmlNetworkAccessManagerFactory>
+#include <QTemporaryFile>
```

Comment:
is this really used ?
missing from the changed lines

Reply:
Nope, more cruft. Removed